pkg/provider/terraform/instance/plugin.go

```diff
@@ -50,7 +53,7 @@ type plugin struct {
 }
 
 // NewTerraformInstancePlugin returns an instance plugin backed by disk files.
-func NewTerraformInstancePlugin(dir string, pollInterval time.Duration, standalone bool) instance.Plugin {
+func NewTerraformInstancePlugin(dir string, pollInterval time.Duration, standalone bool, bootstrapGrpSpecStr, bootstrapInstID string) instance.Plugin {
```

Comment:
@chungers The group spec needs to match https://github.com/docker/infrakit/blob/master/pkg/spi/group/spi.go#L31-L42

For example, see https://github.com/kaufers/infrakit/blob/3c2436308bcc7424030fc2be542bdb715c346bf7/pkg/provider/terraform/instance/plugin_test.go#L2521-L2532

As a concrete example, I've been using:

```
{
  "ID": "managers",
  "Properties": {
    "Allocation": {
      "LogicalIDs": [
        {{ range $i, $_ := loop (int (var "/cluster/swarm/manager/size")) }}
        {{ if ne $i 0 }},{{end}}{{ quote (cat "mgr" (add1 $i) | nospace) }}
        {{ end }}
      ]
    },
    "Instance": {
      {
        "Plugin": "instance-terraform",
        "Properties": {
          "resource": {
            "ibm_compute_vm_instance": {
              "host": {
    "Flavor": {
      "Plugin": "flavor-swarm/manager",
        "Properties": {
        "InitScriptTemplateURL": "{{ $managerInit }}",
        "DockerRestartCommand": "service docker restart",
        "SwarmJoinIP": "{{ var "/cluster/swarm/join/ip" }}",
        "Docker" : {
          "Host" : "unix:///var/run/docker.sock"
        }
      }
    }
  }
}
```